rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -25,6 +25,7 @@
 import co.rsk.peg.*;
 import org.bouncycastle.util.encoders.Hex;
 import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
```

Comment:
Unused, please check this before submitting the PR

Reply:
Seems that there were some issues with my IntelliJ. I had to restart it and invalidate the cache to make it works right again. However, I'll be more cautious with this next time.